### PR TITLE
Hit Ophan endpoint for analytics

### PR DESCRIPTION
**Pull request by oilnam**
Stuff we're monitoring so far:

### Headlines to localized fronts
Capi query: `https://content.guardianapis.com/us?&api-key=test&show-fields=byline,headline&tag=type/article,-tone/minutebyminute,tone/news&show-editors-picks=true`

Url tracked: `https://www.theguardian.com/us`

### Headlines for a topic or a section
Capi query: `https://content.guardianapis.com/us-news/us-politics?&api-key=test&show-fields=byline,headline&tag=type/article,-tone/minutebyminute,tone/news&page-size=3&page=1`

Url tracked: `https://www.theguardian.com/us-news/us-politics`

### Localized opinions
Capi query: `https://content.guardianapis.com/us/commentisfree?&api-key=test&show-fields=byline,headline&tag=type/article,-tone/minutebyminute,tone/comment&page-size=3&page=1`

Url tracked: `https://www.theguardian.com/us/commentisfree`

### Sport opinion (topic)
Capi query: `https://content.guardianapis.com/us/sport?&api-key=test&show-fields=byline,headline&tag=type/article,-tone/minutebyminute,tone/comment&page-size=3&page=1`

Url tracked: `https://www.theguardian.com/us/sport`

### Reviews
Capi query: `https://content.guardianapis.com/search?api-key=test&page-size=3&page=1&tag=tone/reviews,film/film&show-fields=standfirst,byline,headline,star-rating&show-blocks=all`

Url tracked: `https://www.theguardian.com/film+tone/reviews`

## Console log:

Podcasts

### CODE REVIEW

**Comment by mchv** [comment]
When conflicts resolved 👍

**Comment by mchv** [comment]
Please rewrite your history to have a single good written commit message.

File: src/intentLogic/yes.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = function () {
     switch (attributes.lastIntent) {
         case 'GetPodcastIntent':
             const podcastDirective = helpers.getPodcastDirective(attributes.podcastUrl, attributes.podcastTitle);
+            console.log("Playing podcast " + attributes.podcastUrl);
```
Comment (tomrf1) [inline src/intentLogic/yes.js]: do we need this?

Comment (oilnam) [inline src/intentLogic/yes.js]: Yes - this is how we log podcasts at the moment. We're planning to do a proper Ophan integration of all podcasts in the near future.

**Comment by tomrf1** [comment]
👍

**Pull request merged**